src/test/runTest.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+pushd `dirname $0`
+export LD_LIBRARY_PATH=/opt/circonus/lib/amd64:/opt/circonus/lib
+
+if command -v luajit || [ ! -z "$LUA_BIN" ]
+then # lua in path or explicitly speficied
+    LUA_BIN="$(command -v luajit)"
+else # search default locations
+    case $(uname) in
+        SunOS)
+            LUA_BIN=${LUA_BIN:-"/opt/circonus/bin/luajit"}
+            ;;
+        Linux)
+            LUA_BIN=${LUA_BIN:-"/usr/local/bin/luajit"}
```

Comment:
I think we should ensure that /opt/circonus/bin is in PATH, to ensure that we use the Circonus luajit when in the context of a Circonus environment.